examples/version-migration/tree-shim/src/model/inventoryList.ts

```diff
@@ -30,9 +37,47 @@ const newTreeFactory = new SharedTreeFactory({
 	forest: ForestType.Reference,
 });
 
+function migrate(legacyTree: LegacySharedTree, newTree: ISharedTree) {
+	// Revert local edits - otherwise we will be eventually inconsistent
+	const edits = legacyTree.edits as EditLog;
+	const localEdits = [...edits.getLocalEdits()].reverse();
+	for (const edit of localEdits) {
+		legacyTree.revert(edit.id);
+	}
```

Comment:
@DLehenbauer @Abe27342 would appreciate a look to make sure this is the sanctioned way to revert all local/in-flight edits to a legacy shared tree.